### PR TITLE
fix(chat): debounce system prompt updating (Issue #757)

### PR DESCRIPTION
**Pull request by IlyaBondar**
- Issue #757

### CODE REVIEW

**Review by Alexander-Kezik — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -134,9 +136,9 @@
     "tailwindcss": "^3.3.5",
     "ts-node": "10.9.1",
     "typescript": "^5.2.2",
+    "uuid": "^9.0.0",
     "vite": "^5.0.0",
-    "vitest": "^1.2.1",
-    "uuid": "^9.0.0"
```
Comment (mikitabut) [inline package.json:139]: return it back - it needed for e2e tests

Comment (IlyaBondar) [reply package.json:139]: @mikitabut , I didn't return anything as you can see. It was just rearranged automatically

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-776.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-776.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by IlyaBondar** [comment]
/deploy-review

>Environment URL: https://chat-ai-dial-chat-pr-776.nightly-test.deltixhub.io
>E2E tests status: success

File: package.json (hunk 1 above)
Comment (mikitabut) [reply package.json:139]: Got it

File: apps/chat/src/components/Chat/ConversationSettings.tsx [hunk 2]
```diff
@@ -142,6 +144,7 @@ export const ConversationSettings = ({
                     prompt={prompt}
                     prompts={prompts}
                     onChangePrompt={onChangePrompt}
+                    debounceChanges={debounceSystenPromptChanges}
```
Comment (mikitabut) [inline apps/chat/src/components/Chat/ConversationSettings.tsx]: typo in debounceSyste**n**PromptChanges

File: apps/chat/src/components/Chat/SystemPrompt.tsx [hunk 3]
```diff
@@ -120,14 +135,6 @@ export const SystemPrompt: FC<Props> = ({
     }
   }, [content]);
 
-  useEffect(() => {
```
Comment (mikitabut) [inline apps/chat/src/components/Chat/SystemPrompt.tsx:123]: Are you sure that this should be removed?

Comment (IlyaBondar) [reply apps/chat/src/components/Chat/SystemPrompt.tsx:123]: yes, I do. I pass this value in `usePromptSelection` hook. See line 69

**Review by mikitabut — APPROVED** [review]

**Pull request merged**